src/containers/TriggerTemplates/TriggerTemplates.js

```diff
@@ -0,0 +1,187 @@
+
+export /* istanbul ignore next */ class TriggerTemplates extends Component {
+  componentDidMount() {
+    this.fetchTriggerTemplates();
+  }
+
+  componentDidUpdate(prevProps) {
+    const { filters, namespace, webSocketConnected } = this.props;
+    const {
+      filters: prevFilters,
+      namespace: prevNamespace,
+      webSocketConnected: prevWebSocketConnected
+    } = prevProps;
+
+    if (
+      !isEqual(filters, prevFilters) ||
+      namespace !== prevNamespace ||
+      (webSocketConnected && prevWebSocketConnected === false)
+    ) {
+      this.fetchTriggerTemplates();
+    }
+  }
+
+  handleAddFilter = labelFilters => {
+    const queryParams = `?${new URLSearchParams({
+      labelSelector: labelFilters
+    }).toString()}`;
+
+    const currentURL = this.props.match.url;
+    const browserURL = currentURL.concat(queryParams);
+    this.props.history.push(browserURL);
+  };
+
+  handleDeleteFilter = filter => {
+    const currentQueryParams = new URLSearchParams(this.props.location.search);
+    const labelFilters = currentQueryParams.getAll('labelSelector');
+    const labelFiltersArray = labelFilters.toString().split(',');
+    const index = labelFiltersArray.indexOf(filter);
+    labelFiltersArray.splice(index, 1);
+
+    const currentURL = this.props.match.url;
+    if (labelFiltersArray.length === 0) {
+      this.props.history.push(currentURL);
+    } else {
+      const newQueryParams = `?${new URLSearchParams({
+        labelSelector: labelFiltersArray
+      }).toString()}`;
+      const browserURL = currentURL.concat(newQueryParams);
+      this.props.history.push(browserURL);
+    }
+  };
+
+  fetchTriggerTemplates() {
+    const { filters, namespace } = this.props;
+    this.props.fetchTriggerTemplates({
+      filters,
+      namespace
+    });
+  }
+
+  render() {
+    const {
+      error,
+      filters,
+      loading,
+      selectedNamespace,
+      triggerTemplates
+    } = this.props;
+
+    const initialHeaders = [
+      { key: 'name', header: 'Name' },
+      { key: 'date', header: 'Date Modified' }
+    ];
+
+    if (selectedNamespace === ALL_NAMESPACES) {
+      initialHeaders.splice(1, 0, { key: 'namespace', header: 'Namespace' });
+    }
+
+    const triggerTemplatesFormatted = triggerTemplates.map(template => ({
+      id: `${template.metadata.namespace}:${template.metadata.name}`,
+      name: template.metadata.name,
+      namespace: template.metadata.namespace,
+      date: (
+        <FormattedDate date={template.metadata.creationTimestamp} relative />
+      )
+    }));
+
+    return (
+      <div>
+        {error && (
+          <InlineNotification
+            kind="error"
+            title="Error:"
+            subtitle={getErrorMessage(error)}
+            iconDescription="Clear Notification"
+            className="notificationComponent"
+            data-testid="errorNotificationComponent"
+            lowContrast
+          />
+        )}
+        <div>
```

Comment:
Does this need to be wrapped in a div or would we be ok with just the h1? Also avoid including unnecessary whitespace inside tags containing text like this, can lead to "interesting" layout issues in some browsers.